deployments/devnet/declarations.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I understand that you need the `devnet.pkl` but are you using all the other deployment and declaration files? Otherwise suggest to move the kakarot_address and proxy_account_class_hash to `example.env` and be rid with the rest.

Reply:
I think the other deployment files can be of good help if someone wants to run the RPC over the test net!

They wouldn't have to come and ping us on telegram to ask for these addresses and class hash.

But if you think removing them seems better then I can do so! Lemme know your views on it!

Reply:
I think the issue is that the [CI](https://github.com/sayajin-labs/kakarot/blob/main/.github/workflows/deployments.yml) from kakarot will re-deploy all contracts every time we push on main. Meaning these addresses could be out of date as soon as a PR is merged, leading users to use incorrect addresses.
We could add this to the README and point users to the [deployments](https://github.com/sayajin-labs/kakarot/actions/workflows/deployments.yml) on kakarot repo.

Reply:
Makes sense!

Got this done -> [link](https://github.com/sayajin-labs/kakarot-rpc/pull/134/commits/1b6451b812f00c16e54fbee491899e16dda350ab)